### PR TITLE
Add linting for frontend and backend

### PR DESCRIPTION
**Pull request by Dynnammo**
#### :tophat: What? Why?
Add linting workflows to run on main branch and PRs to ease the review process

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #20 ?

#### Testing
*Describe the best way to test or validate your PR.*

#### :clipboard: Checklist
⚠️  No tests suites for now ⚠️
:rotating_light: Please review the [guidelines for contributing](https://github.com/mawoka-myblock/ClassQuiz/blob/master/CONTRIBUTING.md) to this repository.

- [ ] :question: ~~**CONSIDER** adding a unit test if your PR resolves an issue.~~
- [ ] :heavy_check_mark: **DO** check open PR's to avoid duplicates.
- [ ] :heavy_check_mark: **DO** keep pull requests small so they can be easily reviewed.
- [ ] :heavy_check_mark: **DO** build locally before pushing.
- [ ] :heavy_check_mark: ~~**DO** make sure tests pass.~~
- [ ] :heavy_check_mark: ~~**DO** add CHANGELOG upgrade notes if required.~~
- [ ] :x:~~**AVOID** breaking the continuous integration build.~~
- [ ] :x:**AVOID** making significant changes to the overall architecture.

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*

:hearts: Thank you!

### CODE REVIEW

**Comment by Dynnammo** [comment]
Seems we have done the same work each one on our side :sweat_smile: . If you think it's irrelevant you can close this PR

File: .github/workflows/backend_lint.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8
```
Comment (mawoka-myblock) [inline .github/workflows/backend_lint.yml:23]: You don't have to install it extra.

File: .github/workflows/backend_lint.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        python -m pip install pipenv
+        # if [ -f Pipfile ]; then pipenv install; fi
```
Comment (mawoka-myblock) [inline .github/workflows/backend_lint.yml:25]: Here, dont run `pipenv install` run `pipenv instal -d` (`-d` for development-dependencies

File: .github/workflows/backend_lint.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --show-source --statistics
```
Comment (mawoka-myblock) [inline .github/workflows/backend_lint.yml:29]: You **may** have to run `pipenv run flake8...` instead of `flake8` directly.

**Review by mawoka-myblock — CHANGES_REQUESTED** [review]
Added comments about flake and pipenv

**Pull request merged**